cep-XXXX.md

```diff
@@ -0,0 +1,265 @@
+# CEP XXXX - Contents of conda packages
+
+<table>
+<tr><td> Title </td><td> CEP XXXX - Contents of conda packages </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sep 27, 2025</td></tr>
+<tr><td> Updated </td><td> Sep 27, 2025</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/133 </td></tr>
+<tr><td> Implementation </td><td> N/A </td></tr>
+<tr><td> Requires </td><td> https://github.com/conda/ceps/pull/82, https://github.com/conda/ceps/pull/113, https://github.com/conda/ceps/pull/124, https://github.com/conda/ceps/pull/132 </td></tr>
+</table>
+
+## Abstract
+
+This CEP describes the contents of conda packages, regardless of how they are compressed and distributed.
+
+## Motivation
+
+The motivation of this CEP is mostly informative, but will also try to clarify some ambiguous details that should be homogenized across existing implementations.
+
+## Specification
+
+An extracted conda package is a directory that MUST at least include an `info/index.json` file. Other important metadata files SHOULD be included under `info/`.
```

Comment:
Later on the document notes that `./info/paths.json` and `./info/hash_input.json` are also required. So, those should probably be mentioned here as well as part of this "MUST at least include"

```suggestion
An extracted conda package is a directory that MUST at least include `info/index.json`, `./info/paths.json` and `./info/hash_input.json` files. Other important metadata files SHOULD be included under `info/`.
```